src/styles/quiz-card/quiz-card.html

```diff
@@ -0,0 +1,21 @@
+<section class="quiz-card">
+  <img src="../../assets/icon-bookmark.png" alt="" class="quiz-card__icon" />
```

Comment:
I'd give this a description, otherwise the bookmark wouldn't be accessible. 
```suggestion
  <img src="../../assets/icon-bookmark.png" alt="Add bookmark" class="quiz-card__icon" />
```